src/Files.App/Filesystem/Archive/ArchiveCreator.cs

```diff
@@ -11,8 +11,15 @@ namespace Files.App.Filesystem.Archive
 {
 	public class ArchiveCreator : IArchiveCreator
 	{
+		private int itemsAmount = 1;
```

Comment:
What's this for?

Reply:
It's needed to calculate a weighted percentage of progress. If you compress multiple folders, each one will report its own progress. Using `itemsAmount` we can report a realistic progress.
To get the actual progress, I use this formula: 

    [oldPercentage + (percentageDelta / itemsAmount)] 

which means that if we are compressing 2 items and we have compressed 50% of the first one, the total progress will be:
   
    [0% + (50% / 2)] = 25%

Reply:
Can you add a comment to the code?